src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -423,6 +422,9 @@ class TezosNodeSenderOperator(override val node: TezosRPCInterface, network: Str
   import com.muquit.libsodiumjna.{SodiumKeyPair, SodiumLibrary, SodiumUtils}
   import TezosNodeOperator._
 
+  /** Type representing Map[String, Any] */
```

Comment:
just out of curiosity - can you describe your approach to the type aliases regarding where to put them? I always tried to put them in some object with `*Types` in name because at the beginning when I join I saw that way being used somewhere in the project regarding case classes. I have no clear preferences, but maybe we should think of extracting models/types to separate packages, or maybe even objects. WDYT?

Reply:
I certainly see your point here, though I didn't get the naming convention before.
I must admit that "Data" never sounds like a great identifier to me, I tend to avoid any generic naming like "manager", "process", "data", "map", "value" as they can mean all and nothing at the same time.
So I might've missed the idea that `DataTypes` meant type definitions for the chain.DataX objects.
I agree that a consistent naming rule like this makes a lot of sense.

On the other hand, I simply moved the type alias out of the main DataTypes object, because after the refactor, it's only used in two specific contexts (i.e. this file and another).
I wanted to somehow "demote" it from being a globally shared type name, to a local shortcut.
The reason behind my choice was to reduce the chances of any-typed definitions to spread in other parts of the code, unless where strictly necessary.

Yet I'm not too strong on this position, so if you find good reasons for it to be restored as a globally shared public type definition, I have no issue to put it back there

Reply:
No, I don't have any good reason to move it. I just wanted to know about your position about this for future as I'm introducing aliases in my PR 😄